runtime/rasdump/dmpagent.c

```diff
@@ -886,8 +886,7 @@ static omr_error_t
 doJavaDump(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
 {
 	J9JavaVM *vm = context->javaVM;
-
-	if (makePath(vm, label) == OMR_ERROR_INTERNAL) {
+	if (makePath(vm, label) == OMR_ERROR_INTERNAL && 0!= strcmp(label, "/STDOUT/") && 0!= strcmp(label, "/STDERR/")) {
```

Comment:
coding style. Adding brackets and spaces, this should be
`if ((makePath(vm, label) == OMR_ERROR_INTERNAL) && (0 != strcmp(label, "/STDOUT/")) && (0 != strcmp(label, "/STDERR/"))) {`

Reply:
Seems makePath() shouldn't be called at all if the label is `/STDOUT/` or `/STDERR/`. Are these names magic? Won't makePath() have side affects and create a directory `/STDOUT` or `/STDERR`?

Reply:
The character `/` is an illegal character for filenames and/or directories. So makePath with the label `/STDOUT/` shouldn't be possible. Regardless, we shouldn't be making a directory, if we want to write to STDOUT/ERR.